### PR TITLE
feat(alert): create new Alert component and deprecate AlertBox

### PR DESCRIPTION
**Pull request by dfvalero**
## Story
https://jira.zopa/browse/ZDS-16

## Preview
https://zopauk.github.io/react-components/#/Components/Atoms/Alert

### CODE REVIEW

File: src/components/atoms/Alert/Alert.tsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+}
+
+interface IAlertElementsBySeverity {
+  info: IAlertElements;
+  alert: IAlertElements;
+  warning: IAlertElements;
+  success: IAlertElements;
+}
+
+const COLORS_BY_SEVERITY: IAlertElementsBySeverity = {
```
Comment (dfvalero) [inline src/components/atoms/Alert/Alert.tsx:27]: We will use proper color constants in v4!

File: src/components/atoms/Alert/Alert.tsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+  position: relative;
+  padding: 8px 16px 8px 16px;
+  background: ${props => COLORS_BY_SEVERITY[props.severity].background};
+  color: ${props => COLORS_BY_SEVERITY[props.severity].text};
+  font-size: 15px;
+  line-height: 20px;
+  font-weight: 400;
+
+  a {
+    color: ${props => COLORS_BY_SEVERITY[props.severity].text} !important;
```
Comment (dfvalero) [inline src/components/atoms/Alert/Alert.tsx:53]: let me know if we can do this in a cleaner way!

File: package.json [hunk 3]
```diff
@@ -61,6 +62,7 @@
     "@types/react": "16.9.25",
     "@types/react-dom": "16.9.5",
     "@types/styled-components": "4.4.3",
+    "@types/util-deprecate": "^1.0.0",
```
Comment (poteirard) [inline package.json:65]: I think if `util-deprecate` it's imported to the production bundle that should go deps not devDeps

File: src/components/atoms/AlertBox/AlertBox.tsx [hunk 4]
```diff
@@ -27,4 +28,7 @@ const AlertBox: FC<IAlertBoxProps> = ({ children, icon, ...rest }) => (
   </Box>
 );
 
-export default AlertBox;
+/**
+ * @deprecated `<AlertBox />` is deprecated. It will soon be removed. Use `<Alert />` instead.
+ */
+export default deprecate(AlertBox, '<AlertBox /> is deprecated. It will soon be removed. Use <Alert /> instead.');
```
Comment (poteirard) [inline src/components/atoms/AlertBox/AlertBox.tsx:34]: ❤️

**Comment by dfvalero** [comment]
Validated by Rob:

Screenshot 2020-04-21 at 13 49 52

**Pull request merged**